android/app/src/main/AndroidManifest.xml

```diff
@@ -22,9 +22,14 @@
             android:theme="@style/SplashTheme"
             android:configChanges="orientation|screenSize|keyboardHidden">
             <intent-filter>
-                <action android:name="android.intent.action.MAIN" />
-
+                <action android:name="android.intent.action.VIEW"/>
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
                 <category android:name="android.intent.category.LAUNCHER" />
+
+                <data
+                    android:scheme="http"
+                    android:host="2017.fossasia.org" />
```

Comment:
@the-dagger 
i tried various ways to provide the url in java file but wasnt successful.
Also i just want to refer you to this url: https://developer.android.com/training/app-indexing/deep-linking.html
Over here they are hardcoding the strings instead of providng the url somewhere. So just wanted to ask is it really necessary and if it is how to provide the url during runtime as i am not able to set data using getIntent.setData() function.